src/apispec/ext/marshmallow/__init__.py

```diff
@@ -196,6 +196,9 @@ def operation_helper(self, operations, **kwargs):
                     operation["parameters"]
                 )
             if self.openapi_version.major >= 3:
+                for callback in operation.get("callbacks", {}).values():
+                    for event in callback.values():
```

Comment:
Actually, we don't support `{$ref: "MyCallback"}` in other places, so let's not support it here. However, we could expect ref as strings, even if we don't support them yet as per #245. So let's check callback is actually a `dict`.

```py
            if self.openapi_version.major >= 3:
                for callback in operation.get("callbacks", {}).values():
                    if isinstance(callback, dict):
                        for path in callback.values():
```

Reply:
It would be convenient if this code block were implemented in a separate method so that support for callbacks can be documented similar to the other OpenAPI objects that are parsed.

Reply:
@Bangertm do you mean add `callback_helper` to the plugin interface and have it called from `APISpec.path`?

Reply:
No just putting that code block in a method called something like `resolve_callback` and then having a docstring similar to the ones in [SchemaResolver](https://github.com/marshmallow-code/apispec/blob/dev/src/apispec/ext/marshmallow/schema_resolver.py). Then there is some user documentation on the fact that users can put a Marshmallow schema in a callback. 

Ideally this method would be part of `SchemaResolver` too, but given the recursion it can't. You were probably right about making it a mixin.